code/drasil-docLang/lib/Drasil/Sections/TraceabilityMandGs.hs

```diff
@@ -22,6 +22,7 @@ import SysInfo.Drasil
 import Language.Drasil
 import Language.Drasil.Chunk.Concept.NamedCombinators as NC
 import Language.Drasil.Sentence.Combinators as S
+import Data.Foldable(foldl')
```

Comment:
```suggestion
import Data.Foldable (foldl')
```

Super minor formatting thing (I'd do this across these import statements), but I don't think this should be blocking